packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -1,46 +1,21 @@
-import { useEffect } from 'react';
 import { RequestLineFragment } from '../api/operations.generated';
 import {
   useTranslation,
   ColumnAlign,
   useColumns,
   GenericColumnKey,
   QuantityUtils,
-  useSortBy,
-  SortBy,
-  zustand,
   getCommentPopoverColumn,
   useFormatNumber,
+  useQueryParamsStore,
 } from '@openmsupply-client/common';
 import { useRequestFields } from '../api';
 
-type Store = {
-  sortBy: SortBy<RequestLineFragment>;
-  setSortBy: (sortBy: SortBy<RequestLineFragment>) => void;
-};
-
-const useStore = zustand<Store>(set => ({
-  sortBy: { key: 'itemName', isDesc: false, direction: 'asc' },
-  setSortBy: (sortBy: SortBy<RequestLineFragment>) =>
-    set(state => ({ ...state, sortBy })),
-}));
-
-const useSharedSortBy = () => {
-  const sharedSortBy = useStore();
-  const { sortBy, onChangeSortBy } = useSortBy<RequestLineFragment>(
-    sharedSortBy.sortBy
-  );
-
-  useEffect(() => {
-    sharedSortBy.setSortBy(sortBy);
-  }, [sortBy]);
-  return { sortBy, onChangeSortBy };
-};
-
 export const useRequestColumns = () => {
   const t = useTranslation('common');
   const { maxMonthsOfStock } = useRequestFields('maxMonthsOfStock');
-  const { sortBy, onChangeSortBy } = useSharedSortBy();
+  const { sort } = useQueryParamsStore();
+  const { sortBy, onChangeSortBy } = sort;
```

Comment:
the whole `sharedSortBy` is no longer needed, as the QueryParamsStore handles it